data/data.go

```diff
@@ -101,36 +106,37 @@ func (d *dataService) ReleaseDB(id, version string) {
 	dbMapSync.Lock()
 	defer dbMapSync.Unlock()
 
-	db := dbMap[versionedID]
-	if db != nil {
-		dbMap[versionedID] = nil
+	dbm := dbMap[versionedID]
+	if dbm.db != nil {
```

Comment:
Should be `dbm!=nil`. It's a nil pointer panic if there's no this version in map.